src/components/elementView/powerView/powerView.tsx

```diff
@@ -260,14 +254,18 @@ export class PowerView extends React.PureComponent<
                 {this.props.asset!.power_connections[i].left_right}
               </td>
             ) : (
-              <td></td>
+              <td />
             )}
             {this.props.asset!.rack &&
             this.props.asset!.rack.is_network_controlled ? (
-              this.state.powerStatus ? (
+              this.props.assetIsDecommissioned || this.props.changePlan ? (
```

Comment:
little bit confusing here but choosing between showing spinner, showing power status, showing connection error message, and showing not pdu controlled message

Reply:
cool -> i trust you that you've tested this, but if you haven't already can you write some test cases that should run through all the different combinations so that we have expected behavior?